erpnext/stock/doctype/landed_cost_voucher/landed_cost_voucher.js

```diff
@@ -261,47 +261,33 @@ erpnext.stock.LandedCostVoucher = erpnext.stock.StockController.extend({
 		$.each(item_total_fields || [], function(i, f) {
 			totals[f] = flt(frappe.utils.sum((me.frm.doc.items || []).map(d => flt(d[f]))));
 		});
-
-		var charges_map = [];
-		var manual_account_heads = new Set;
-		var idx = 0;
-		$.each(me.frm.doc.taxes || [], function(i, tax) {
-			if (tax.base_amount) {
-				var based_on = frappe.scrub(tax.distribution_criteria);
-
-				if(based_on == "manual") {
-					manual_account_heads.add(cstr(tax.account_head));
-				} else {
-					if(!totals[based_on]) {
-						frappe.throw(__("Cannot distribute by {0} because total {0} is 0", [tax.distribution_criteria]));
+		$.each(me.frm.doc.items || [], function(i, item) {
+			item.item_tax_detail = {}
+			let item_manual_distribution = JSON.parse(item.manual_distribution || '{}');
+			$.each(me.frm.doc.taxes || [], function(i, tax) {
+				item.item_tax_detail[tax.name] = 0;
+				let distribution_amount;
+				let distribution_based_on = frappe.model.scrub(tax.distribution_criteria);
+				if (distribution_based_on == 'manual') {
+					distribution_amount = flt(item_manual_distribution[tax.account_head]);
+				}
+				else {
+					if (!totals[distribution_based_on]){
+						frappe.throw(_("Cannot distribute by {0} because total {0} is 0").format(tax.distribution_criteria))
```

Comment:
Use `__` instead of `_` for transalate
`.format` doesn't work in client side. See how it was being used before